src/main/java/com/espro/flink/consul/jobgraph/ConsulSubmittedJobGraphStore.java

```diff
@@ -86,6 +91,41 @@ public JobGraph recoverJobGraph(JobID jobId) throws Exception {
         return getStateHandle(jobId).retrieveState();
     }
 
+    @Override
+    public CompletableFuture<Void> localCleanupAsync(JobID jobId, Executor executor) {
```

Comment:
Add tests for `localCleanupAsync` and `globalCleanupAsync`

Reply:
done